### DISCUSSION TITLE
Size of icon option

### DISCUSSION THREAD

**Discussion by Sodacan259** [extesy/hoverzoom#655] [Ideas]
Excellent extension.

Websites that have a mixture of small and large image formats mean that zooming is redundant for the larger images.
An option to set the maximum size of the icon to be zoomed would be great.

So if I set dimensions 300x300px as a preference, any icon that is larger than those bounds would not be zoomed, as it's not necessary.
e.g. icon of 250x302px or 301x250px or 301x301px would not be zoomed, but 300x300px and smaller would.
Meaning that the smaller images/icons on the page are zoomed, but larger ones are not.

This would also be helpful as a catch-em-all for websites that have images larger than the preference - without having to add the website to the blacklist.

**Comment by GrosPoulet** [comment]
Thank you for using HZ+

I already added some new options to my development version of HZ+ that may fit your needs:
![image](https://user-images.githubusercontent.com/23529041/104758086-29842300-575e-11eb-9c0e-1a4a076484af.png)
I'll try to push all that new stuff to GitHub next week. stay tuned :stuck_out_tongue_winking_eye:

**Comment by Sodacan259** [comment]
Oh! That's cool.
I'll keep a lookout.
Thanks!

Get Outlook for Android<https://aka.ms/ghei36>

________________________________
